### DISCUSSION TITLE
[@types/intercom-web] IntercomSettings is missing api_base

### DISCUSSION THREAD

**Discussion by maphe** [DefinitelyTyped/DefinitelyTyped#59222] [Issues with a @types package]
The Intercom documentation requires the tag on page to be like 
```ts
<script>
  window.intercomSettings = {
    api_base: "https://api-iam.intercom.io",
    app_id: "[app_id]"
  };
</script>
```

However @types/intercom-web, which declares `window.intercomSettings` is missing the `api_base` attribute.

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
I just opened #59226

**Reply by maphe** [reply]
Cool, thanks, that was quick.

Do you know by any chance if it can be turned into a more proper module? I'm having a hard time importing it without errors
```
import 'intercom-web';

Error: Module not found: Error: Can't resolve 'intercom-web'
```
or 
```
import * as Intercom from 'intercom-web';

TS2306: File '.../node_modules/@types/intercom-web/index.d.ts' is not a module.

```

**Reply by MysteryBlokHed** [reply]
It looks like intercom-web defines global values instead of being imported like a module. Having the `@types/intercom-web` package installed should be all you need for types to work. Try typing `Intercom` or `intercomSettings` without importing anything and seeing if your IDE autocompletes it.

If that doesn't work, adding this in your TypeScript source file might help:

```typescript
/// <reference types="intercom-web" />
```

**Reply by maphe** [reply]
Ok thanks, ended up putting in the tsconfig file:
```json
"compilerOptions": {
    "types": [
      "intercom-web"
    ]
  },
```
Looks like the IDE is happier now. Thanks again 👍
